lib/rubocop/cop/lint/erb_new_arguments.rb

```diff
@@ -6,18 +6,18 @@ module Lint
       #
       # This cop emulates the following Ruby warnings in Ruby 2.6.
       #
-      # % cat example.rb
-      # ERB.new('hi', nil, '-', '@output_buffer')
-      # % ruby -rerb example.rb
-      # example.rb:1: warning: Passing safe_level with the 2nd argument of
-      # ERB.new is deprecated. Do not use it, and specify other arguments as
-      # keyword arguments.
-      # example.rb:1: warning: Passing trim_mode with the 3rd argument of
-      # ERB.new is deprecated. Use keyword argument like
-      # ERB.new(str, trim_mode:...) instead.
-      # example.rb:1: warning: Passing eoutvar with the 4th argument of ERB.new
-      # is deprecated. Use keyword argument like ERB.new(str, eoutvar: ...)
-      # instead.
+      #   [source,console]
+      #   ----
+      #   % cat example.rb
+      #   ERB.new('hi', nil, '-', '@output_buffer')
+      #   % ruby -rerb example.rb
+      #   example.rb:1: warning: Passing safe_level with the 2nd argument of ERB.new is
+      #   deprecated. Do not use it, and specify other arguments as keyword arguments.
+      #   example.rb:1: warning: Passing trim_mode with the 3rd argument of ERB.new is
+      #   deprecated. Use keyword argument like ERB.new(str, trim_mode:...) instead.
+      #   example.rb:1: warning: Passing eoutvar with the 4th argument of ERB.new is
+      #   deprecated. Use keyword argument like ERB.new(str, eoutvar: ...) instead.
+      #   ----
```

Comment:
Sorry, the indentations I showed was redundant.

```suggestion
      # [source,console]
      # ----
      # % cat example.rb
      # ERB.new('hi', nil, '-', '@output_buffer')
      # % ruby -rerb example.rb
      # example.rb:1: warning: Passing safe_level with the 2nd argument of ERB.new is
      # deprecated. Do not use it, and specify other arguments as keyword arguments.
      # example.rb:1: warning: Passing trim_mode with the 3rd argument of ERB.new is
      # deprecated. Use keyword argument like ERB.new(str, trim_mode:...) instead.
      # example.rb:1: warning: Passing eoutvar with the 4th argument of ERB.new is
      # deprecated. Use keyword argument like ERB.new(str, eoutvar: ...) instead.
      # ----
```